homeassistant/components/traccar/__init__.py

```diff
@@ -1 +1,109 @@
-"""The traccar component."""
+"""Support for Traccar."""
+import logging
+
+import voluptuous as vol
+from aiohttp import web
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import HTTP_UNPROCESSABLE_ENTITY, HTTP_OK, CONF_WEBHOOK_ID
+from homeassistant.helpers import config_entry_flow
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.components.device_tracker import DOMAIN as DEVICE_TRACKER
+from .const import (
+    ATTR_ACCURACY,
+    ATTR_ALTITUDE,
+    ATTR_BATTERY,
+    ATTR_BEARING,
+    ATTR_ID,
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    ATTR_SPEED,
+    ATTR_TIMESTAMP,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+TRACKER_UPDATE = "{}_tracker_update".format(DOMAIN)
+
+
+DEFAULT_ACCURACY = 200
+DEFAULT_BATTERY = -1
+
+
+def _id(value: str) -> str:
+    """Coerce id by removing '-'."""
+    return value.replace("-", "")
+
+
+WEBHOOK_SCHEMA = vol.Schema(
+    {
+        vol.Required(ATTR_ID): _id,
```

Comment:
Maybe:
```py
vol.All(cv.string, _id)
```

Reply:
I force-pushed but what did you mean with this?

Reply:
It might good to validate that it's a string before we do the string replacement.